course/meetings/eeb-asher-martin-2018-02-14.md

```diff
@@ -0,0 +1,122 @@
+        have enough assignment points handed out before the drop deadline for them
+        to know how they stand in the class.
+        -   Also wait with the group work until the drop deadline has past, we don't
+            want them to drop out from the groups.
+    -   Typically there are around 20-25 4th year students in the research course.
+        Could be a good idea to schedule for fall and advertise for these students.
+        -   They could send emails about this during summer.
+-   The same room for Tue and Thu would be great. To deal with less IT issues.
+-   Have a formal presentation for the final project
+    -   Angle as an end of semester symposium for the students (since no one took up
+        the offer to submit officially this year, we will make a presentation a
+        lecture component for last year).
+-   More updates during the project work, have a set of questions that the student
+    answer at the beginning of each class to make sure they share their struggles
+    and surceases with the other groups
+-   We are free to reuse any assignments we want, just take them down from the
+    homepage.
+
+
+<a id="org9e6dffd"></a>
+
+## Teaching setup
+
+-   There will probably be around 3-4 of us and 0-1 grad students from EEB.
+-   We could either be TAs as last year or have a setup where 1-2 of us are course
+    (co)-instructors and then 1-2 TAs.
+    -   Course instructor positions have to be posted, but are usually written so that
+        it is difficult for not difficultly.
+    -   However, Asher prefers someone with past experience to be the course
+        instructor, so not Ahmed as we originally though =(
+        -   Although, I think if we have two co-instructors, maybe it could be
+            Ahmed together with someone from last fall.
+    -   In addition to this we could always include guest lecturers from UofTCoders
+        if we need more help.
+    -   It is up to us how the pay will be divided, but usually TA help is not given
+        to course under 25 students so can't demand too much.
+
+
+<a id="org90ef1d8"></a>
+
+## Content redesign
+
+-   Important for next year is to make sure that the analysis techniques required
+    for the group work are all covered during the class ours.
+-   In the student feedback, many wished they had additional lecture time focused
+    on mixed effects modelling.
+-   Asher thinks it's ok to keep the diversity of data sets (and not make it only
+    time series for example).
+    -   They get more ownership over research question, but it does depend on the
+        objective of the course.
+-   A good idea could be to select 2-3 types data sets (where "type" refers to
+    that they would require distinct analysis methods) with 1-2 data sets each.
+    -   Make sure that each of these analyses techniques are covered during class.
+-   To accommodate this deeper focus on the modelling techniques, the part that
+    Madeleine taught last year will be modified.
```

Comment:
The nice thing is that the course won't be limited by the fact that we had "Theoretical ecology" in the title!